bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/models/form/FieldType.java

```diff
@@ -35,7 +35,10 @@ public enum FieldType {
     PANEL("panel"),
     FORM("form"),
     CHECKBOX_GROUP("checkbox-group"),
-    IMAGE("image");
+    IMAGE("image"),
+    TELEPHONE("tel"),
```

Comment:
We would want to support password and range also in eds, but may be not in cc for the starting. Either we should the sling model or give a static mapping for such components.

Reply:
password is already being worked upon as part of this PR, https://github.com/adobe/aem-core-forms-components/pull/1316

Reply:
Without having it defined now would mean that it would default to text-input. Which is compatible for now until the password core component is done.